lib/ansible/module_utils/vmware.py

```diff
@@ -302,6 +302,7 @@ def gather_vm_facts(content, vm):
         'snapshots': [],
         'current_snapshot': None,
         'vnc': {},
+        '_moId': vm._moId
```

Comment:
This does align well with the other names.

Reply:
Yes, I agree but the property returned by vSphere schema is just like this. So wanted to match things here as well.